Mathlib/Logic/Equiv/Fin.lean

```diff
@@ -30,13 +30,6 @@ def finZeroEquiv' : Fin 0 ≃ PEmpty.{u} :=
 def finOneEquiv : Fin 1 ≃ Unit :=
   Equiv.equivPUnit _
 
-/-- Equivalence between `Fin 2` and `Bool`. -/
-def finTwoEquiv : Fin 2 ≃ Bool where
-  toFun := ![false, true]
-  invFun b := b.casesOn 0 1
-  left_inv := Fin.forall_fin_two.2 <| by simp
-  right_inv := Bool.forall_bool.2 <| by simp
-
 /-!
 ### Tuples
```

Comment:
One reasonable split might be to move this to `Logic/Equiv/Fin/Tuple` or `Fin/Tuple/Equiv`.